addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -534,6 +534,15 @@ var SnippetEditor = Widget.extend({
                 } else {
                     $optionsSection.append(option.$el);
                 }
+                if (option.$el !== undefined) {
+                    option.$el.tooltip({
+                        selector: 'we-title',
+                        placement: 'bottom',
+                        title: function () {
+                            return (this.scrollWidth > this.offsetWidth) ? this.textContent : '';
```

Comment:
- There can be html content in the we-title, so textContent would break it
- Don't use `this` but instead name the received parameter "el" and use it. Using `this` as the received element is counter-intuitive and it is a source of error.